### PR TITLE
Update microsoft-teams from 1.5.00.11505 to 1.5.00.9159

### PR DESCRIPTION
**Pull request by miccal**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask/pull/122642 should not have been merged -- this version is not shown by 'livecheck` because it is an experimental build of a universal-version of Microsoft Teams, and is not ready for release.

See https://feedbackportal.microsoft.com/feedback/idea/df49aed2-1573-ec11-8943-0022482f9e90?ocid=usoc_TWITTER_M365_spl100002925765523, in particular:

> As we have shared, we are working on a native Mac client for M1. We understand that our users are eager for a native M1 build, and we are excited to make it a reality in the near future!   We ask that our users please not use unvalidated, unsupported versions. Recently some sites have published articles about a Microsoft Teams Beta release of a Universal Binary application that runs natively on both M1 and Intel based processors. The information in these articles is inaccurate, and they have no backing from Microsoft. In the articles, there are links to early builds on our CDN that are for internal use only. We can confirm we are working on delivering a Universal Binary offering of Teams, and the CDN link in these articles does contain a downloadable version. However, we discourage users from trying this version (and any other versions that do not come from official Microsoft distribution channels), as it is still in the early phases of testing, fixing, and validation. Thank you.

The version shown inside the Microsoft Teams app is ` 1.5.00.11505-E`, where the "E" means "experimental".

File: Casks/microsoft-teams.rb [hunk 1]
```diff
@@ -17,10 +17,7 @@
 
   pkg "Teams_osx.pkg"
 
-  uninstall pkgutil:   [
-    "com.microsoft.teams",
-    "com.microsoft.MSTeamsAudioDevice",
-  ],
+  uninstall pkgutil:   "com.microsoft.MSTeamsAudioDevice",
```
Comment (SMillerDev) [inline Casks/microsoft-teams.rb:20]: Why change this one?

Comment (miccal) [reply Casks/microsoft-teams.rb]: I removed the wrong one.

Comment (miccal) [reply Casks/microsoft-teams.rb]: @SMillerDev can we no longer approve our own PR's?

Comment (SMillerDev) [reply Casks/microsoft-teams.rb]: You could never approve your own PRs. But if you are in a hurry to merge your own PR you can use ` brew pr-publish --tap homebrew/cask --workflow publish-commit-casks.yml <id>`

**Review by SMillerDev — APPROVED** [review]

File: Casks/microsoft-teams.rb (hunk 1 above)
Comment (miccal) [reply Casks/microsoft-teams.rb]: Thanks -- I assume `<id>` here means `miccal` in my case?

Comment (SMillerDev) [reply Casks/microsoft-teams.rb]: No, the PR number actually. That wasn't great naming on my part

**Pull request merged**